crates/core/src/commands/check.rs

```diff
@@ -447,7 +447,9 @@ fn check_packs_list(be: &impl ReadBackend, mut packs: HashMap<PackId, u32>) -> R
         .map_err(RusticErrorKind::Backend)?
     {
         match packs.remove(&PackId::from(id)) {
-            None => warn!("pack {id} not referenced in index. Can be a parallel backup job. To repair: 'rustic repair index'."),
+            None => {
+                warn!("(hot) pack {id} not referenced as tree pack in index, but as data pack.");
```

Comment:
this is currently used for both hot packs and for normal (maybe cold) packs. So changing this this way makes it correct for hot packs, but wrong in the standard case.

Reply:
Ah! How would we differentiate in `None` between hot and cold packs, then I can implement it.

Reply:
I copied the check_pack function and now handle checking of hot packs differently from normal packs. Also added a test about data packs contained in the hot repo.

Reply:
Is duplicating the function and code the only good way to handle it, currently? Do you think we will implement different handling (checking) for hot and cold files in the future, so we could reuse this?

Reply:
It is already different. In the hot pack check we now additionally check against the total pack list to give another error in the specific situation the user had...